apiml-security-common/src/main/java/org/zowe/apiml/security/common/error/ErrorType.java

```diff
@@ -14,6 +14,7 @@
  * Enum of error types
  * binding error keys and default error messages
  */
+// TODO Find better message for BAD_CREDENTIALS. For now we make the response consistent with the non-modulith
```

Comment:
Is this TODO valid?

Reply:
Yes, I guess we need to change the response message to better one. I.e. `org.zowe.apiml.security.login.invalidInput` would be more proper message, but that would cause inconsistency with the non-modulith.